packages/@okta/i18n/src/properties/login_cs.properties

```diff
@@ -533,10 +533,10 @@ primaryauth.submit = Prihl\u00E1sit
 primaryauth.newUser.tooltip=Toto je va\u0161e prvn\u00ED pripojen\u00ED k {0} z tohoto prohl\u00ED\u017Eece
 primaryauth.newUser.tooltip.close=Zavr\u00EDt
 oktaVerify.description = Pro p\u0159\u00EDstup k tomuto zdroji vy\u017Eaduje va\u0161e organizace p\u0159ihl\u00E1\u0161en\u00ED pomoc\u00ED va\u0161eho za\u0159\u00EDzen\u00ED.
-oktaVerify.button = P\u0159ihlaste se pomoc\u00ED Okta Verify na tomto za\u0159\u00EDzen\u00ED
+oktaVerify.button = P\u0159ihl\u00E1sit se pomoc\u00ED Okta FastPass
 
 # IDP Discovery
-idpDiscovery.email.placeholder = Email
+idpDiscovery.email.placeholder = E-mail
```

Comment:
We're starting to converge on using the hyphenated spelling for this language, but there appears to be a few [instances](https://github.com/okta/okta-signin-widget/blob/c4f0536b85b316cf7f38fdc3fdf2b6b1bd5b49cc/packages/%40okta/i18n/src/properties/login_cs.properties#L665) where we aren't.

Reply:
I'll have the team scrub the compound construct in the next delivery. It's not a show-stopping change.